private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go

```diff
@@ -73,8 +73,10 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.Username,
 		usernameFlagName,
 		"",
-		"The username to use. This command prompts for a username by default",
+		"The username to use.",
 	)
+	_ = flagSet.MarkDeprecated(usernameFlagName, "This flag no longer has any effect")
```

Comment:
Change to "This flag is no longer needed as the username is automatically derived from the token"